app/main.py

```diff
@@ -1,13 +1,13 @@
-# from models import Actor
-# from managers import ActorManager
-#
-# if __name__ == "__main__":
-#     Actor.objects = ActorManager()
-#
-#     Actor.objects.create(first_name="Emma", last_name="Watson")
-#     Actor.objects.create(first_name="Daniel", last_name="Radclife")
-#     print(Actor.objects.all())
-#     Actor.objects.update(2, "Daniel", "Radcliffe")
-#     print(Actor.objects.all())
-#     Actor.objects.delete(1)
-#     print(Actor.objects.all())
+from models import Actor
+from managers import ActorManager
+
+if __name__ == "__main__":
+    Actor.objects = ActorManager()
+
+    Actor.objects.create(first_name="Emma", last_name="Watson")
+    Actor.objects.create(first_name="Daniel", last_name="Radcliffe")
+    print(Actor.objects.all())
+    Actor.objects.update(2, "Daniel", "Radcliffe")
```

Comment:
The `update` method is called with three parameters: `2`, `"Daniel"`, and `"Radcliffe"`. However, the `update` method in `ActorManager` expects an `id` as the first parameter and the new `first_name` and `last_name` as the second and third parameters. Ensure that the `id` is an integer and corresponds to an existing actor entry in the database.